apps/studio/src/components/tableinfo/TableSchema.vue

```diff
@@ -262,8 +266,20 @@ export default Vue.extend({
           editable: this.isCellEditable.bind(this, 'alterColumn'),
           formatter: this.cellFormatter,
           cellEdited: this.cellEdited,
-          editor: vueEditor(NullableInputEditorVue)
+          editor: vueEditor(NullableInputEditorVue),
+          minWidth: 90,
         }),
+        {
+          title: 'Comment',
+          field: 'comment',
```

Comment:
@azmy60 this is great! One thing we need to change though -- not all databases support `comment` on columns. You'll need to check `this.disabledFeatures?.comments`, similar for how we check for `extra` on line 261 above.

If you want to see where this is specified, take a look at src/lib/dialects/sqlite.ts <- you can see comments are not supported by Sqlite.